src/components/ProfileDropdown.jsx

```diff
@@ -66,66 +65,79 @@ const useStyles = makeStyles((theme) => ({
+    return (
+      <>
+        <MenuItem onClick={task} className={classes.menuListItem}>
+          <ListItemIcon>{icon}</ListItemIcon>
+          <ListItemText primary={text} />
+        </MenuItem>
+      </>
+    );
+  }
 
-  const handleClose = () => {
-    setDropMenu(null);
+  const display = {
+    0: {
+      manageProfile: (
+        <>
+          {EXPERIMENTAL_VERSION ? null : (
+            <a href={manageProfileLink}>Manage Your Profile</a>
+          )}
+        </>
+      ),
+      navigation: (
+        <>
+          {EXPERIMENTAL_VERSION ? null : (
+            <>
+              {createMenuItem('New', <NewIcon fontSize="medium" />)}
+              {createMenuItem('Saved', <BookmarkIcon fontSize="medium" />)}
+              {createMenuItem('Applied', <DoneIcon fontSize="medium" />)}
+            </>
+          )}
+        </>
+      ),
+    },
   };
 
   return (
-    <Container>
-      <Avatar src={photo} onClick={handleClick} className={classes.large} />
+    <>
+      <Avatar src={photo} onClick={handleClick} />
       <StyledMenu
         anchorEl={dropMenu}
         keepMounted
         open={Boolean(dropMenu)}
         onClose={handleClose}>
-        <Grid container spacing={2} className={classes.root}>
-          <Grid item xs={3} sm={3} className={classes.profilePic}>
+        <Grid container spacing={2} className={classes.gridRoot}>
+          <Grid item className={classes.profilePic}>
             <Avatar src={photo} className={classes.medium} />
           </Grid>
-          <Grid item xs={9} sm={9}>
+          <Grid>
             <Typography variant="h6" component="h4">
               {name}
             </Typography>
             <Typography component="h6" gutterBottom>
               {email}
             </Typography>
-            <a href="/home">Manage Your Profile</a>
+            {display[0].manageProfile}
           </Grid>
         </Grid>
-        <Divider />
-        <MenuItem className={classes.menuList}>
-          <ListItemIcon>
-            <NewIcon fontSize="medium" />
-          </ListItemIcon>
-          <ListItemText primary="New" />
-        </MenuItem>
-        <MenuItem className={classes.menuList}>
-          <ListItemIcon>
-            <BookmarkIcon fontSize="medium" />
-          </ListItemIcon>
-          <ListItemText primary="Saved" />
-        </MenuItem>
-        <MenuItem className={classes.menuList}>
-          <ListItemIcon>
-            <DoneIcon fontSize="medium" />
-          </ListItemIcon>
-          <ListItemText primary="Applied" />
-        </MenuItem>
-        <Divider />
-        <MenuItem onClick={signOut} className={classes.menuList}>
-          <ListItemIcon>
-            <ExitToAppIcon fontSize="medium" />
-          </ListItemIcon>
-          <ListItemText primary="Logout" />
-        </MenuItem>
+        <Divider className={classes.dividerSpacing} />
+        {display[0].navigation}
+
+        {EXPERIMENTAL_VERSION ? null : (
```

Comment:
Use the [logical && operator](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator) instead.

Reply:
Ok